charts/stable/cs2/Chart.yaml

```diff
@@ -17,7 +17,13 @@ dependencies:
     tags: []
     import-values: []
 deprecated: false
-description: A custom SteamCMD chart that runs CS2.
+description: |
```

Comment:
Add to the docs, don't make random changes. Its a chart description, not a how to description.